packages/playground/src/types/index.ts

```diff
@@ -57,7 +57,27 @@ export enum ProjectName {
   Qvm = "Qvm",
   Umbrel = "Umbrel",
 }
-
+export const DeploymentTitle: { [key: string]: string } = {
+  Kubernetes: "Kubernetes Clusters",
+  CapRover: "CapRover Clusters",
+  Discourse: "Discourse Instances",
+  Funkwhale: "Funkwhale Instances",
+  Mastodon: "Mastodon Instances",
+  Mattermost: "Mattermost Instances",
+  Owncloud: "Owncloud Instances",
+  Peertube: "Peertube Instances",
+  Subsquid: "Subsquid Instances",
+  Taiga: "Taiga Instances",
+  Wordpress: "Wordpress Instances",
+  TFhubValidator: "TFhubValidator Instances",
```

Comment:
We removed the validator no?

Reply:
should we have only the current active weblets ? 
as qvm should be removed to

Reply:
QVM should also be hidden too